src/appkit.cs

```diff
@@ -1232,6 +1260,16 @@ interface NSBezierPath : NSCoding, NSCopying {
 
 		[Export ("flatness")]
 		nfloat Flatness { get; set; }
+
+		[Mac (10,13)]
+		[Export ("appendBezierPathWithCGGlyph:inFont:")]
+		void AppendBezierPathWithCGGlyph (CGGlyph glyph, NSFont font);
```

Comment:
So the problem here is that we already have an AppendBezierPath variant that takes a uint, font and a bunch of older version that use WithFoo.

If I make this one different I think it will be confusing, and I can't convert them all without hitting the collision. :(